lib/workers/repository/reconfigure/comment.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { logger, partial, platform } from '../../../../test/util';
+import { getConfig } from '../../../config/defaults';
+import { GlobalConfig } from '../../../config/global';
+import type { RenovateConfig } from '../../../config/types';
+import type { PackageFile } from '../../../modules/manager/types';
+import type { Pr } from '../../../modules/platform';
+import type { BranchConfig } from '../../types';
+import { ensureReconfigurePrComment, getConfigDesc } from './comment';
+
+describe('workers/repository/reconfigure/comment', () => {
+  describe('ensureReconfigurePrComment()', () => {
+    let config: RenovateConfig;
+    let packageFiles: Record<string, PackageFile[]>;
+    let branches: BranchConfig[];
+    const reconfigureBranch = 'renovate/reconfigure';
+    const reconfigurePr = partial<Pr>({ number: 1 });
+
+    beforeEach(() => {
+      config = {
+        ...getConfig(),
```

Comment:
do we really need the full config here? (perf)

this will transitive load all platforms /managers /datasources and versionings.